static/lang/action-en.json

```diff
@@ -62,6 +62,17 @@
                 },
                 "Title": "Arcane Slam"
             },
+            "ArrestAFall": {
+                "Description": "<p>You attempt your choice of an Acrobatics check or Reflex save to slow your fall. The DC is typically 15, but it might be higher due to air turbulence or other circumstances.</p><p><strong>Success</strong> You take no damage from the fall.</p>",
+                "Notes": {
+                    "success": "<strong>Success</strong> You take no damage from the fall."
+                },
+                "Title": "Arrest A Fall"
+            },
+            "AvertGaze": {
+                "Description": "<p>You avert your gaze from danger, such as a medusa's gaze. You gain a +2 circumstance bonus to saves against visual abilities that require you to look at a creature or object, such as a medusa's petrifying gaze. Your gaze remains averted until the start of your next turn.</p>",
+                "Title": "Avert Gaze"
+            },
```

Comment:
These don't appear to have macros associated with them.

Reply:
When I first mentioned updating these, @nikolaj-a said the following:

> Feel free to add the `"Description"` to any of them, though.
> At least any of the basic or specialty basic ones.
> They'll get converted eventually and then they need that text.
> Adding in the description text is probably the most time-consuming part of the conversion 🙈
> It needs to be formatted properly etc.

Based on that, I added these two actions to the file for future-proofing.